lang/de_DE.json

```diff
@@ -0,0 +1,416 @@
+    "Reset Connect API Key": "Verbindungs-API-Schlüssel zurücksetzen",
+    "Reset Game Progress": "Spielfortschritt zurücksetzen",
+    "Reset Progress": "Fortschritt zurücksetzen",
+    "Reset Web API Key": "Web-API-Schlüssel zurücksetzen",
+    "Reset avatar!": "Profilbild wurde zurücksetzt!",
+    "Resetting progress...": "Fortschritt wird zurückgesetzt...",
+    "Resetting...": "Wird zurückgesetzt...",
+    "Responsibility for the content of external links (to web pages of third parties) lies solely with the operators of the linked pages. No violations were evident to us at the time of linking. Should any legal infringement become known to us, we will remove the respective link immediately.": "Für den Inhalt externer Links (zu Webseiten Dritter) sind ausschließlich deren Betreiber verantwortlich. Zum Zeitpunkt der Verlinkung waren für uns keine Rechtsverstöße ersichtlich. Bei Bekanntwerden von Rechtsverstößen werden wir den entsprechenden Link umgehend entfernen.",
+    "Restored {{gameTitle}}!": "{{gameTitle}} wurde wiederhergestellt!",
+    "Restoring...": "Wird wiederhergestellt...",
+    "RetroAchievements may revise these terms of use at any time without notice. By using its services, you agree to the current version of these Terms of Use, all applicable laws and regulations, and agree that you are responsible for compliance with any applicable local laws.": "RetroAchievements kann diese Nutzungsbedingungen jederzeit und ohne Vorankündigung überarbeiten. Durch die Nutzung der Dienste erklärst du dich sich mit der aktuellen Version dieser Nutzungsbedingungen, allen geltenden Gesetzen und Vorschriften einverstanden und stimmst zu, dass du für die Einhaltung aller geltenden lokalen Gesetze verantwortlich sind.",
+    "RetroAchievements.org does not condone or supply any copyright-protected ROMs to be used in conjunction with the supplied emulators.": "RetroAchievements.org duldet oder liefert keine urheberrechtlich geschützten ROMs, die in Verbindung mit den mitgelieferten Emulatoren verwendet werden können.",
+    "RetroAchievements.org's Terms of Use are subject to change.": "Die Nutzungsbedingungen von RetroAchievements.org können sich jederzeit ändern.",
+    "RetroPoints: A measurement of rarity and estimated difficulty.": "RetroPunkte: Ein Maß für die Seltenheit und den geschätzten Schwierigkeitsgrad.",
+    "Revision": "Revision",
+    "Search games": "Spiele suchen",
+    "Search games...": "Spiele werden gesucht...",
+    "See more": "Siehe mehr",
+    "Select a game": "Wähle ein Spiel",
+    "Select an achievement": "Wähle eine Errungenschaft",
+    "Settings": "Einstellungen",
+    "Show absolute dates on forum posts": "Absolute Daten in Forenbeiträgen anzeigen",
+    "Site Awards": "Website-Auszeichnungen",
+    "Site notifications": "Website-Benachrichtigungen",
+    "Someone follows me": "Jemand folgt mir",
+    "Something went wrong.": "Etwas ist schiefgelaufen.",
+    "Sort order": "Sortiere Reihenfolge",
```

Comment:
Is "Sortierreihenfolge" more accurate?

Reply:
From the context of the use it would make more sense to use "Sortiere Reihenfolge" instead since this describes the action while "Sortierreihenfolge" describes an already sorted list with certain criterias